LocalSettings.php

```diff
@@ -4203,6 +4203,16 @@
 			'bot' => array(
 				'editinterface' => true,
 			),
+			'user' => array(
```

Comment:
@Reception123 for some reason my above comment is all screwed up. It is supposed to read like this:

by default these permissions are assigned to the 'all' group, but I'm revoking them from the 'all' group and assigning them to the 'user' group instead (because the 'all' group would override the "banned" group)